What's next after CONTRIBUTING.md?

Simran1604 opened the pull request:
Report of What's next after CONTRIBUTING.md? added.

Simran1604 committed:
What's next after CONTRIBUTING.md?

Report of What's next after CONTRIBUTING.md? added.

deprov447 committed:
Minor fixes

deprov447 commented:
@Simran1604 Please add [front-matter](https://www.clairecodes.com/blog/2021-02-19-creating-custom-front-matter-in-hugo-files/) to file (the initial 5 lines of [this](https://raw.githubusercontent.com/p-society/psoc-website-new/master/content/Events/others/cybersec.md) for example). 
Additionally fix the spelling mistakes

>  You can always check your **pushed** code's preview [here](https://deploy-preview-16--dev-psoc.netlify.app/)

Simran1604 committed:
Update Whats_next_after_CONTRIBUTING.md

Simran1604 committed:
Update Whats_next_after_CONTRIBUTING.md

deprov447 committed:
Front matter fix

deprov447 committed:
Major structural changes

- Created a new Events/"hacktoberfest.md"
- Removed Resources.md (obsolete)
- Minor spelling fixes
